site/pages/disruption-detail/disruption-detail.test.tsx

```diff
@@ -302,5 +302,31 @@ describe("pages", () => {
 
             unmount();
         });
+        it("should render correctly with inputs and no errors when disruption has no consequences", () => {
```

Comment:
Add a snapshot to check the text "Add a consequence" when number of consequences are zero

Reply:
I believe that is showing in the snapshot

Reply:
made tests more specific